lib/app.ex

```diff
@@ -115,7 +113,20 @@ defmodule App do
 
     File.write("./README.md", docs)
 
-    File.write!("./snippets.json", snippet_json)
+    elixir_snippet_json =
```

Comment:
Will make this a bit nicer once we figure out if this solves the problem of scoping to the root.